### DISCUSSION TITLE
【新人指南】邮箱信息

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#2873] [其他]
test
![29093112nr7z](https://github.com/user-attachments/assets/a1fa0f45-1eb9-4b3d-a51d-ba65f89c3c3f)

当你在绳网发布帖子或者评论后，你的邮箱📮可能会接受到大量的信息。

_一觉醒来以为自己被网暴了_<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

关闭单条Discussion的订阅
![Screenshot_2024_0819_010057](https://github.com/user-attachments/assets/4d49de0e-d33e-45e0-8d01-8084b4d3b6de)
讨论帖子下面就能找到

**Comment by ITManCHINA** [comment]
其实只需要关闭单条Discussion的订阅就行，毕竟Github的邮箱提醒有时候还是有必要的😂
![image](https://github.com/user-attachments/assets/5733e295-2ae9-414d-a237-dfcf64f41aed)

**Reply by jsishsjsj** [reply]
有道理(๑•̀㉨•́ฅ✧
